concepts/type-checking/about.md

```diff
@@ -0,0 +1,144 @@
+# About
+
+Knowning what type an object has is often very important for code to run smoothly and without errors.
+
+Javascript has several ways to check the type of an object.
+
+```exercism/note
+Javascript's type checking mechanisms are always soomewhat unreliable.
+
+For true safety with types, you should probably use TypeScript, a language that builds on JavaScript, but with the type syntax of a static-typed language.
```

Comment:
```suggestion
For true safety with types, you should probably use TypeScript, a language that builds on JavaScript, but with the type syntax of a static-typed language.
```
```suggestion
For better type safety and stronger types, you should probably use TypeScript, a language that builds on JavaScript, but with the type syntax of a static-typed language.
```

TypeScript (and Flow) actually don't give *true* safety, but it's "good enough".